src/System.Management.Automation/engine/remoting/commands/newrunspacecommand.cs

```diff
@@ -1136,6 +1150,28 @@ private List<RemoteRunspace> CreateRunspacesForSSHHostHashParameterSet()
             return remoteRunspaces;
         }
 
+        /// <summary>
+        /// Helper method to create remote runspace based on UseWindowsPowerShell parameter set.
+        /// </summary>
+        /// <returns>Remote runspace that was created.</returns>
+        private List<RemoteRunspace> CreateRunspacesForUseWindowsPowerShellParameterSet()
```

Comment:
Is there a reason this method returns a `List<RemoteRunspace>` rather than `RemoteRunspace` or `IEnumerable<RemoteRunspace>` or `IReadOnlyList<RemoteRunspace>`?

Reply:
Consistency with naming in existing semantically-similar code. There are 7 other `List<RemoteRunspace> CreateRunspacesFor*ParameterSet()` methods.